[ALS-4151] - Clone SWB example repos on workspace init

Luke-Sikina opened the pull request:
- Add clone script
- Call script during bootstrap
- All that again, but for Windows
- Update docs

srpiatt commented on docs/docs/user_guide/sidebar/common/workspaces/accessing_a_workspace.md:
```diff
@@ -80,3 +80,14 @@ Connection to workspace is restricted to specific CIDR block.
+Service Workbench and using the PIC-SURE API. You can find these repositories in the following
+locations, depending on instance type:
+
+- **EC2 Linux**: `/home/ec2-user/repos/`
```
Let's check with out BSAs if the folder name `repos` conveys enough to the end user.

Luke-Sikina replied:
Simran: `example_workflows`

srpiatt commented on docs/docs/user_guide/sidebar/common/workspaces/accessing_a_workspace.md:
```diff
@@ -80,3 +80,14 @@ Connection to workspace is restricted to specific CIDR block.
+### Example Repositories
+All workspaces come with some examples to get you started with accessing data using the 
+Service Workbench and using the PIC-SURE API. You can find these repositories in the following
+locations, depending on instance type:
```
Love this note! Short and sweet, but helpful. :)

srpiatt commented on main/solution/post-deployment/config/environment-files/clone_repos.sh:
Is the code that calls this script in a different PR?

Luke-Sikina replied:
I added the changes to get_bootstrap after you made this comment in an elaborate attempt to make you look foolish.

srpiatt commented on main/solution/post-deployment/config/environment-files/clone_repos.sh:
```diff
@@ -0,0 +1,46 @@
+sudo yum update -y
+sudo yum install -y git
+
+echo "Done installing git. Cloning repos: " >> /var/log/clone.log
```
Maybe piping/copying the entire console log and error output to /var/log/clone.log would be most helpful in the future, should some misc line in this script fails. (Probably wherever you call this script would be the best place.) This level of logging is good, but I am just overly suspicious of all code lines-- I've had stupid little things fail and it takes forever to figure out the source cause without a full end to end execution log.

srpiatt reviewed (approved):
Can merge after verifying the repos name is okay for our end users and wider review in office hours.

Luke-Sikina commented on main/solution/post-deployment/config/environment-files/clone_repos.sh:
```diff
@@ -0,0 +1,46 @@
+# symlinks inside symlinks doesn't seem to work, so we're just going to cp these to the sample notebooks
+if [ -d "/home/ec2-user/sample-notebooks" ]
+then
+    sudo cp -r $repo_dir/* /home/ec2-user/sample-notebooks/
```
Should we remove other notebooks since they don't work?

Luke-Sikina replied:
Simran: delete them all, but send document that list and give to bsas

Luke-Sikina committed:
[ALS-4151] - Clone SWB example repos on workspace init #87

- Add clone script
- Call script during bootstrap
- All that again, but for Windows
- Update docs
